t/ui/12-needle-edit.t

```diff
@@ -328,12 +333,36 @@ subtest 'Needle editor layout' => sub {
     is @{decode_utf8_json(element_prop('needleeditor_textarea'))->{area}}, 2, 'exclude areas always present';
     $driver->find_element_by_xpath('//input[@value="inst-timezone"]')->click();
     is @{decode_utf8_json(element_prop('needleeditor_textarea'))->{area}}, 2, 'no duplicated exclude areas present';
+
+    # select needle to take image from
+    $driver->execute_script("document.getElementById('image_select').selectedIndex = 1; loadBackground();");
+    is element_prop('image_select'), 'inst-timezone-text', 'selected needle to take image from';
+    sleep 0.1 until $driver->execute_script('return nEditor.bgImage.complete;');
```

Comment:
I consider 100ms quite long for unit tests. Can we use .01?

Reply:
Ok, I can change that.